schemas/schemas/adminQuery.json

```diff
@@ -8,6 +8,10 @@
         "description"
     ],
     "properties": {
+        "comment": {
+            "description": "Arbitrary comment for reference purposes.",
+            "type": ["string", "array", "object"]
```

Comment:
See discussion thread above for whether we should use `"type": "string"` or this more complicated type everywhere.

Reply:
propagated to all in d80069e1c